### ISSUE TITLE
change_dates() feature is flip-flopping start and end_date values

### ISSUE THREAD

**Issue by tpo5** [CDCgov/Rnssp#13]
When providing start and end dates for change_dates(), the dates are sometimes updated backwards in the ESSENCE API URL. This causes the API call not to return any records. When examining the URL, it is subbing the start date value in for the end date and vice versa if startDate appears first in the URL. 

```r

library(Rnssp)

# Does not work because startDate appears first in the URL
url1 <- "https://essence2.syndromicsurveillance.org/nssp_essence/api/dataDetails/csv?percentParam=noPercent&datasource=va_erccdd&detector=nodetectordetector&syndrome=resp&timeResolution=weekly&hasBeenE=1&ccddCategoryFreeText=ISNULL,or,ISBLANK,or,%5E,andnot,%5E;CDC%20Broad%20Acute%20Respiratory%20DD%20v1;%5E&aqtTarget=TimeSeries&startDate=30Jul23&endDate=30Jul23"

change_dates(url1, start_date = "2022-09-01", end_date = "2022-09-02")

# Works because endDate appears first in the URL
url2 <- "https://essence2.syndromicsurveillance.org/nssp_essence/api/dataDetails/csv?percentParam=noPercent&datasource=va_erccdd&detector=nodetectordetector&syndrome=resp&timeResolution=weekly&hasBeenE=1&ccddCategoryFreeText=ISNULL,or,ISBLANK,or,%5E,andnot,%5E;CDC%20Broad%20Acute%20Respiratory%20DD%20v1;%5E&aqtTarget=TimeSeries&endDate=30Jul23&startDate=30Jul23"

change_dates(url2, start_date = "2022-09-01", end_date = "2022-09-02")

```

**Comment by rosericazondekon** [comment]
Thank you @tpo5  for reporting this bug in the `change_dates()` function. A patch will be pushed shortly to the development version of the package. The fix will ultimately be part of the next version (`Rnssp v0.2.1`) of the package when we release it.

Thank you so much for your contribution.

**Comment by rosericazondekon** [comment]
A fix has been pushed to the [dev branch](https://github.com/CDCgov/Rnssp/blob/dev/R/utility.R). Feel free to check it. Again, this bug fix will be effective as of `Rnssp v0.2.1` when we release it.
Again, thank you @tpo5  for reporting this bug.
